[New Concept]: Symbols (#1560)
exercism/ruby: exercises/concept/port-palermo/.docs/introduction.md

# About

[Symbols][symbols] are named identifiers that can be used to refer to a value.
Symbols are created through a symbol literal, which is by prefixing a name with a `:` character, e.g. `:foo`.

> They also allow for being written as strings, e.g. `:"foo"`, which allows for example spaces in the name.

Reviewer: ```suggestion
They also allow for being written with quotes, e.g. `:"a foo"`, which allows for example spaces in the name.
```

This also demonstrates the benefit of using strings with the symbol creation, allowing for a space in the symbol, reinforcing the example shown in the IRB session below.  (Well, it could be an IRB session, anyway...)

```ruby
:foo # => :foo
:"foo boo" # => :"foo boo"

> ```

Reviewer: ~~~suggestion
```irb
>> :foo
=> :foo
>> :"foo boo"
=> :"foo boo"
```
~~~
